### PR TITLE
DDF-1910: updates to gh-pages for v.2.9.0 javadoc

### PR DESCRIPTION
**Pull request by ricklarsen**
#### What does this PR do?
Adds the version 2.9.0 javadoc to the gh-pages branch

#### Who is reviewing it?

@jrnorth 
#### Choose 2 committers to review/merge the PR (please choose ONLY two committers from below, delete the rest).

@jlcsmith 
@jaymcnallie 
#### How should this be tested?
Verify links work

#### What are the relevant tickets?

https://codice.atlassian.net/browse/DDF-1910

#### Checklist:

- [x] Documentation Updated
- [ ] Update / Add Unit Tests
- [ ] Update / Add Integration Tests

### CODE REVIEW

File: Documentation-versions.html [hunk 1]
```diff
@@ -47,61 +47,57 @@
...
                     <p>API documentation of DDF</p>
-                </div>
+                    <div class="doc-buttons">
+                        <a class="btn btn-primary" href="api/index.html" target="_blank">Javadoc &raquo;</a>
+                    </div>
+                </div> -->
             </div>
 
 
             <div class="row">
```
Comment (jrnorth) [inline Documentation-versions.html]: Could this `div` be deleted now?

**Comment by jrnorth** [comment]
Documentation links work as expected and point to the `2.9.0` docs.

File: Documentation-versions.html (hunk 1 above)
Comment (ricklarsen) [inline Documentation-versions.html]: yes

**Comment by jlcsmith** [comment]
LGTM

**Comment by jaymcnallie** [comment]
LGTM

**Pull request merged**